shared/domains/chelonia/utils.js

```diff
@@ -92,7 +92,7 @@ const validateActionPermissions = (signingKey: SPKey, state: Object, opT: string
     // available for us to see. In this case, we ignore the missing key.
     // For incoming messages, we must check permissions and a missing
     // key means no permissions.
-    if (!innerSigningKey && direction === 'outgoing') return true
+    if (!innerSigningKey && msg._direction === 'outgoing') return true
```

Comment:
Shouldn't we use `msg.direction()` instead of `_direction` to access this?

Reply:
I figured it's fine since SPMessage is part of Chelonia, and this saves a function call.